internal/sirius/get_deputy_details.go

```diff
@@ -6,11 +6,15 @@ import (
 	"net/http"
 )
 
-type executiveCaseManager struct {
+type ExecutiveCaseManager struct {
 	EcmId   int    `json:"id"`
 	EcmName string `json:"displayName"`
 }
 
+type ExecutiveCaseManagerOutgoing struct {
+	EcmId int
```

Comment:
I believe you can stick json annotations on outgoing structs as well as incoming. `json:"ecmId"` should allow you to use a lowercase property in the PHP.